### PR TITLE
AAP 2.1 release notes

### PR DESCRIPTION
**Pull request by neal-timpe**

### CODE REVIEW

**Review by thaumos — APPROVED** [review]
LGTM.

I'd like to re-think this for 2.2 onwards.  We should probably coordinate with marketing and other teams earlier in the cycle.

File: release-notes/topics/aap-21.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[[aap-2.1-intro]]
+= Ansible Automation Platform 2.1
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+.Enhancements
+
+* Added separate execution capacity and update your cluster without downtime with Automation Mesh that connects the cluster with a flexible communication method using receptor
```
Comment (cbudz) [inline release-notes/topics/aap-21.adoc:8]: I recommend rewording this sentence to lead with Automation mesh. So something like "Automation mesh introduces a separated execution capacity ...."

File: release-notes/topics/aap-21.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* The platform installer can deploy a Highly Available (HA) Automation Hub cluster
+* Automation Hub connects instances to external Red Hat SSO instances
```
Comment (cbudz) [inline release-notes/topics/aap-21.adoc:10]: This also allows for the installer to deploy RH SSO, may be worth saying.

**Review by cbudz — CHANGES_REQUESTED** [review]

**Comment by neal-timpe** [comment]
@cbudz Knocked out those comments. Ready for another look.

**Review by cbudz — APPROVED** [review]

**Pull request merged**